tests/foreman/destructive/test_ldap_authentication.py

```diff
@@ -411,7 +407,9 @@ def test_login_failure_rhsso_user_if_internal_user_exist(
         login=username,
         password=settings.rhsso.rhsso_password,
     ).create()
-    external_rhsso_user = create_new_rhsso_user(client_id, username=username)
+    external_rhsso_user = default_sso_host.create_new_rhsso_user(
+        default_sso_host.get_rhsso_client_id(), username=username
```

Comment:
just remove this `default_sso_host.get_rhsso_client_id()` as not required here, and move that as first line to get the latest token